release-notes/MediaOps/MediaOps_1.4.2.md

```diff
@@ -29,6 +29,10 @@ In the Scheduling app, the default sort order on the *Ops Board* page has been i
 
 This enhancement also prevents unexpected changes to the order of the jobs on the *Ops Board* page when an action is executed that influences the timing of a job.
 
+#### Scheduling: Nodes Overview page removed from Edit job panel [ID 44085]
+
+In the Scheduling app, the *Nodes Overview* page has been removed from the *Edit job* panel, as this page was not used.
```

Comment:
Would say that maximizing the nodes table functionality is removed from the job edit panel, as it was not working and not used:
<img width="1678" height="978" alt="image" src="https://github.com/user-attachments/assets/d2f39ce6-a741-43d4-88d2-136cdca5a797" />